src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 60px;
+  height: 60px;
+  display: block;
+}
+
+.video {
+  margin-right: 0;
+  box-sizing: border-box;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
+  margin: 0 auto;
+  border-radius: 8px;
+  position: absolute;
+  top: 56px;
+}
+
+.nav__link:hover:hover {
+  color: #00acdc;
+}
+
+.form {
+  list-style-type: none;
+  display: flex;
+  align-items: baseline;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  margin-right: 20px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, style elements using classes. This makes your CSS more modular and avoids unintended side effects.